Bottom: 5138322446780f0ddd893f64f3913bdb7ed21287
Top:    b30c7413474bb117e19f1345ab7a9750ff9dddc1
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:01:39 +0000

Refresh of polygon-debug

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 4b38532..364d875 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1504,7 +1504,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
                               copper_appearance,
                               NULL));
 
-
     break;
   }
